VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -30,28 +33,35 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textColor="@color/edx_grayscale_neutral_x_dark"
-            android:textSize="@dimen/edx_small" />
+            android:textSize="@dimen/edx_small"
+            android:maxLines="2"
+            android:ellipsize="end"
+            tools:text="This is a post title. It is a very long title to demonstrate that post titles will be truncated when they are too long."/>
 
         <LinearLayout
             android:orientation="horizontal"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content">
 
-            <org.edx.mobile.third_party.iconify.IconView
-                android:id="@+id/discussion_post_pin_or_following_icon"
-                style="@style/icon_view_standard_size_style"
-                android:layout_marginEnd="3dp"
-                android:layout_marginRight="3dp"
-                app:iconColor="@color/edx_grayscale_neutral_light"
-                android:visibility="gone" />
+            <TextView
+                android:id="@+id/discussion_post_pin_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:drawablePadding="3dp"
```

Comment:
For some reason I was thinking *sp* wouldn't make sense because we are trying to scale horizontally, not vertically. But now I don't know why I thought that, *sp* would work fine.

> I don't think we need to automate the padding scaling to match the font

Why not? If the icon and text are twice as big, they would appear too close together unless the padding scales as well.